# creates a planner
string plannertype
---
uint32 plannerid
